src/calculateRentalCost.js

```diff
@@ -5,6 +5,21 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  let amount = 0;
+
+  for (let i = 0; i <= days - 1; i++) {
+    amount += 40;
+  }
+
+  if (days >= 7) {
```

Comment:
2. [Checklist] Don't use literals in expressions (HARDCODE). Instead use constants explaining these values.

for this and some other lines